nautobot_data_validation_engine/jobs.py

```diff
@@ -0,0 +1,72 @@
+"""Jobs for nautobot_data_validation_engine."""
+
+from django.apps import apps as global_apps
+
+from nautobot.extras.models import GitRepository
+from nautobot.extras.jobs import Job, MultiChoiceVar, BooleanVar
+
+from nautobot_data_validation_engine.custom_validators import get_data_compliance_rules_map, get_classes_from_git_repo
+
+
+def get_data_compliance_rules():
+    """Generate a list of Audit Ruleset classes that exist from the registry."""
+    validators = []
+    for rule_sets in get_data_compliance_rules_map().values():
+        validators.extend(rule_sets)
+    return validators
+
+
+def get_choices():
+    """Get choices from registry."""
+    choices = []
+    for ruleset_class in get_data_compliance_rules():
+        choices.append((ruleset_class.__name__, ruleset_class.__name__))
+    for repo in GitRepository.objects.all():
+        if "nautobot_data_validation_engine.data_compliance_rules" in repo.provided_contents:
+            for compliance_class in get_classes_from_git_repo(repo):
+                choices.append((compliance_class.__name__, compliance_class.__name__))
+
+    choices.sort()
+    return choices
+
+
+class RunRegisteredDataComplianceRules(Job):
+    """Run the validate function on all registered DataComplianceRule classes."""
+
+    selected_classes = MultiChoiceVar(
+        choices=get_choices,
+        label="Select Data Compliance Classes",
+        required=False,
+        description="Not selecting any classes will run all classes listed.",
+    )
+    override_enforce = BooleanVar(
```

Comment:
What I had in mind was if we didn't override the enforce variable and the job failed, then it would be a clear indicator that some of the data still isn't clean.  

That said, as I sat here making sure I didn't have any other use cases, having a clear indicator of data being clean or not is kinda the whole point of the various views of the `DataCompliance` objects... so this is redundant.  Gonna remove this unless there's another use case I'm not thinking of?

Reply:
Removed this override selection.